lib/luatest_server.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, *args, **kwargs):
     def execute(self, server):
         """Execute test by luatest command
 
-        Execute `luatest -c --no-clean --verbose <name>_test.lua --output tap`
-        command. Disable capture mode and deletion of the var directory.
-        Provide a verbose output in the tap format. Extend the command by
-        `--pattern <pattern>` if the corresponding option is provided.
+        Execute `luatest -c --no-clean --log <path> --verbose <name>_test.lua --output tap`
```

Comment:
I expect that the option will be at the same place according to your code. So it should go after `--output tap`.